Services/Certificate/classes/class.ilCertificateGUI.php

```diff
@@ -556,7 +556,7 @@ private function saveCertificate(ilPropertyFormGUI $form, array $form_fields, $o
 					}
 				}
 				if($backgroundImagePath === '') {
-					if ($_POST['background_delete']) {
+                    if ($_POST['background_delete'] || $previousCertificateTemplate->getBackgroundImagePath() === '') {
```

Comment:
Looks like wrong indentation (tabs/spaces).

Reply:
Fixed it. Thanks!